components/flipper.py

```diff
@@ -2,18 +2,28 @@
 
 
 class Flipper:
+    #this flips a bucket with an arm that has a spinning motor
+    
 
     flipper_motor = wpilib.Talon
+    # a talon controls a motor
```

Comment:
I think documenting what Talons do is unnecessary, and the name is pretty self-explanatory to start with, so this comment is probably unnecessary.